modnet/featurizers/featurizers.py

```diff
@@ -96,6 +100,7 @@ def _fit_apply_featurizers(
         featurizers: Iterable[BaseFeaturizer],
         column: str,
         fit_to_df: bool = True,
+        mode: str = "single",
```

Comment:
As `multi` is the default mode of the class, could we also make it default here ?

Reply:
Yes, my bad

```suggestion
        mode: str = "multi",
```